src/raw-value.ts

```diff
@@ -285,6 +274,15 @@ export function reverseProxyFactory(env: MembraneBroker) {
     ReflectSetPrototypeOf(ReverseProxyHandler.prototype, null);
 
     function getRawValue(sec: SecureValue): RawValue {
+        if (isNullOrUndefined(sec)) {
```

Comment:
reorganizing the check here, using the information to decide what to do to avoid double checking based on the type.